bh_modules/mdcodeinline.py

```diff
@@ -0,0 +1,22 @@
+"""
+BracketHighlighter.
+
+Copyright (c) 2013 - 2015 Isaac Muse <isaacmuse@gmail.com>
+License: MIT
+"""
+
+
+def post_match(view, name, style, first, second, center, bfr, threshold):
+    """Ensure that backticks inside the inline are not highlighted."""
+    # ensure this is only used for inline markdown
+    if name != "mdcodeinline":
```

Comment:
Yes it is not necessary, but I added it to indicate and ensure it cannot used for something else. In contrast to [pyquotes](https://github.com/facelessuser/BracketHighlighter/blob/master/bh_modules/pyquotes.py), which could be used in a lot of cases.
In addition I don't have to worry about other preconditions, e.g. a bracket inside an inline code could result in `end` being less than `first.begin`.